### DISCUSSION TITLE
AOSC Website Redesign

### DISCUSSION THREAD

**Discussion by euotiniel** [angolaosc/.github#9] [Ideas]
### Visão Geral
O objetivo deste projeto é redesenhar o website da Angola Open Source Community para torná-lo mais atraente e funcional. Nossa equipe fez um esforço para simplificar o design e melhorar a navegação para garantir que os visitantes tenham uma experiência agradável enquanto exploram nosso site.

```veja a versão atual: ``` https://aosc.social/

### Pré-visualização

O modelo completo pode ser encontrado [clicando nesse link](https://www.figma.com/proto/9Lup4f63Uc1WVPQQlyupNR/AOSC?node-id=1-2)

![Captura de Ecrã (450)](https://github.com/angolaosc/.github/assets/93485038/c1d2dd57-4e80-4879-b1f8-28f39968b605)

```Nota:``` A versão atual do design no Figma está focada apenas no "desenho do site", confesso que não foram levados em conta todos os conseitos e recurso oferecidos pelo Figma.

### Feedback e Contribuições

Valorizamos suas opiniões e sugestões. Se você deseja fornecer feedback ou contribuir para este projeto, deixe um comentário nesta ```issues``` ou entre em contacto [comigo](https://github.com/euotiniel).

### Autor

```Nome:``` [Otoniel Emanuel](https://euotiniel.com)
```E-mail:``` otonielemanuel21@gmail.com

**Comment by ssukuakueche** [comment]
Eu gostei bastante.🥳

**Comment by ssukuakueche** [comment]
@antonio-pedro99 Acredito que deves verificar isto.

**Comment by makiessedev** [comment]
`Vamos a isso`
